tests/test_manual_connect.c

```diff
@@ -117,14 +151,15 @@ void test_connect_with_duo_passcodeInPassword(void** unused)
     snowflake_term(sf);
 }
 
-int main(void)
-{
+
+int main(void) {
```

Comment:
let's apply the same formatting rules that we have in other places in the library

Reply:
Done.